### PR TITLE
[pythnet] print Pyth keys on startup

### PR DESCRIPTION
**Pull request by swimricky**
add log to print pyth related keys on validator start up

### CODE REVIEW

File: runtime/src/bank.rs [hunk 1]
```diff
@@ -2747,6 +2747,40 @@ impl Bank {
...
+            ),
+            (
+                "WORMHOLE_PID",
+                Pubkey::new_from_array(pythnet::WORMHOLE_PID),
+            ),
+        ];
+        let pyth_pubkeys: Vec<(&str, std::result::Result<Pubkey, AccumulatorUpdateError>)> =
+            pyth_keys
+                .iter()
+                .map(|(k, d)| (*k, self.env_pubkey_or(k, *d)))
```
Comment (ali-bahjati) [inline runtime/src/bank.rs:2779]: We don't read the accumulator address from environment though :?

File: runtime/src/bank.rs [hunk 2]
```diff
@@ -2747,6 +2747,40 @@ impl Bank {
             .unwrap_or(default))
     }
 
+    /// Get all pyth related pubkeys from environment variables
+    /// or return default if the variable is not set.
+    pub fn get_pyth_keys(
```
Comment (ali-behjati) [inline runtime/src/bank.rs]: I would rename this to `get_accumulator_keys`

File: core/src/validator.rs [hunk 3]
```diff
@@ -583,6 +583,18 @@ impl Validator {
                 info!("Hard forks: {:?}", hard_forks);
             }
         }
+        let pyth_keys = bank.get_pyth_keys();
+        for (key_name, pk_res) in pyth_keys {
+            match pk_res {
+                Ok(pk) => {
+                    info!("{}: {}", key_name, pk);
```
Comment (ali-behjati) [inline core/src/validator.rs]: It's good to prefix this with "Accumulator"

File: runtime/src/bank.rs (hunk 2 above)
Comment (swimricky) [reply runtime/src/bank.rs]: fixed thanks

File: core/src/validator.rs (hunk 3 above)
Comment (swimricky) [reply core/src/validator.rs]: fixed thanks

**Review by ali-behjati — APPROVED** [review]

**Pull request merged**